apps/server/src/services/integration-service/OscIntegration.ts

```diff
@@ -113,6 +110,44 @@ export class OscIntegration implements IIntegration<OscSubscription, OSCSettings
     }
   }
 
+  private stringToOSCArgs(argsString): ArgumentType[] {
+
+    // NOTE: regex taken from https://stackoverflow.com/questions/4031900/split-a-string-by-whitespace-keeping-quoted-segments-allowing-escaped-quotes
+    const parseRegex = /[\w.]+|"(?:\\"|[^"])+"/g;
+
+    const parsedArguments = argsString.match(parseRegex).map((argString: string) => {
+      const argAsNum = Number(argString);
+
+      // NOTE: number like: 1 2.0 33333
+      if (!Number.isNaN(argAsNum)) {
+        if (argString.includes('.')) {
+          return {
+            type: 'f',
+            value: argAsNum,
+          };
+        } else {
+          return {
+            type: 'i',
+            value: argAsNum,
+          };
+        }
+      }
```

Comment:
suggestion: I see that the only real difference is the type, it could make it easier to read if that was made explicit- Something like:

```ts
   const isValidNumber = !Number.isNaN(argAsNum)
    if (isValidNumber) {
      const argAsNum = Number(argString);
      const type = argString.includes('.') ? 'f' : 'i';
      return { type, value: argAsNum };
    }
```